targets/UnrealMarketplacePlugin/source/PlayFab/Source/PlayFabCommon/Private/PlayFabCommmonUtils.cpp.ejs

```diff
@@ -0,0 +1,46 @@
+﻿<%- copyright %>
```

Comment:
My concerns with this file are that it may not compile for any platform except windows.
Have you tested this at all with other platforms?
It doesn't have to work, but it must still compile.